pkg/config/path.go

```diff
@@ -152,11 +153,19 @@ type backendPath struct {
 	Configuration dictPath `path:".configuration"`
 }
 
+type providerPath struct {
```

Comment:
please update tests for `path.go` as well

Reply:
Done.